pkg/types/args_test.go

```diff
@@ -0,0 +1,113 @@
+package types_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestArgs(t *testing.T) {
```

Comment:
How exactly does that work? I've been manually running the tests using plain `go test github.com/appc/cni/pkg/types` so far.